Add new Roborock integration documentation (#26520)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

---
title: Roborock

> description: Instructions on how to integrate your Roborock vacuum with home assistant

Reviewer: ```suggestion
description: Instructions on how to integrate Roborock vacuums into Home Assistant
```
I'm not sure where/if this string is used, but this should likely be in plural form?

ha_category:
  - Vacuum
ha_iot_class: Cloud Polling
ha_release: 2023.4
ha_config_flow: true
ha_codeowners:
  - '@humbertogontijo'
  - '@Lash-L'
ha_domain: roborock
ha_platforms:
  - vacuum
ha_integration_type: integration
---

> The `roborock` integration allows you to control your Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum.

Reviewer 2: ```suggestion
The Roborock integration allows you to control your Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum.
```

> The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while continuing to use the Roborock app. In contrast to [Xiaomi Miio](https://www.home-assistant.io/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access and device configuration using the Roborock app.

Reviewer 2: ```suggestion
The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while continuing to use the Roborock app.

In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access and device configuration using the Roborock app.
```

> The Roborock integration allows you to control your Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while syncing everything from the Roborock app. The [Xiaomi Miio](https://www.home-assistant.io/integrations/xiaomi_miio/) integration can also control Roborock vacuums, but you must use the Mi home app instead of the Roborock app. This integration is currently cloud based, while the Xiaomi integration has the option of local control.

Reviewer: ```suggestion
The Roborock integration allows you to control your Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while keeping using the Roborock app. In contrast to [Xiaomi Miio](https://www.home-assistant.io/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access and device configuration using the Roborock app.
```
Just a suggestion to reword this a bit to clearly point out the strengths & weaknesses compared to the existing integration.

I have learned that it is also a good idea not to add todo-like (word "currently") information to the documentation :-)

---
title: Roborock
description: Instructions on how to integrate Roborock vacuums into Home Assistant
ha_category:
  - Vacuum
ha_iot_class: Local Polling
ha_release: 2023.5
ha_config_flow: true
ha_codeowners:
  - '@humbertogontijo'
  - '@Lash-L'
ha_domain: roborock
ha_platforms:
  - vacuum
ha_integration_type: integration
---

The Roborock integration allows you to control your [Roborock](https://us.roborock.com/pages/robot-vacuum-cleaner) vacuum while continuing to use the Roborock app.

> In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access to set up and device configuration using the Roborock app.
> 
> Once you login with your Roborock account, the integration will automatically discover your Roborock devices, and get the needed information to communicate locally with them. Make sure your Home Assistant instance can communicate with the local IP of your device. We recommend setting a static IP for your Roborock Vacuum to help prevent future issues.

Reviewer 2: ```suggestion
In contrast to [Xiaomi Miio](/integrations/xiaomi_miio/) integration, this integration provides more complete support but requires cloud access to set up and device configuration using the Roborock app.

Once you log in with your Roborock account, the integration will automatically discover your Roborock devices and get the needed information to communicate locally with them. Please ensure your Home Assistant instance can communicate with the local IP of your device. We recommend setting a static IP for your Roborock Vacuum to help prevent future issues.
```
